Scripts/DCS-BIOS/lib/modules/aircraft_modules/Ka-50.lua

```diff
@@ -0,0 +1,906 @@
+module("Ka-50", package.seeall)
+
+local Module = require("Module")
+
+--- @class Ka_50 : Module
+local Ka_50 = Module:new("Ka-50", 0x1800, { "Ka-50", "Ka-50_3" })
+
+--overhaul by WarLord v2.0b
+
+-- Hide Stick Arg# 1005
+
+local function Ka_50:definePushButtonLed(msg, arg_number, category, description)
+	local function round(n)
+		return n % 1 >= 0.5 and math.ceil(n) or math.floor(n)
+	end
+	local value = moduleBeingDefined.memoryMap:allocateInt({
+		maxValue = 1,
+	})
+	assert(value.shiftBy ~= nil)
+	moduleBeingDefined.exportHooks[#moduleBeingDefined.exportHooks + 1] = function(dev0)
+		value:setValue(round(dev0:get_argument_value(arg_number) * 10) % 2)
+	end
```

Comment:
AP_BANK_HOLD_LED was "1" in ctrlref
arg:330 returned 0.10000000149012.

	local function round(n)
		return n % 1 >= 0.5 and math.ceil(n) or math.floor(n)
	end
	
	local value = 0.10000000149012
	
	print(round(value * 10) % 2)  => "1"

Reply:
I would suggest to leave it. At this stage it is the migration that has the focus, refactoring modules where we are not sure of the consequences is not IMO wanted.

Reply:
> floor(value * 10) enough?

unlikely, since I'm guessing that value could maybe be 0.30 in which case we would still want to output 1. Same with 0.0999999...

> I would suggest to leave it. At this stage it is the migration that has the focus, refactoring modules where we are not sure of the consequences is not IMO wanted.

you're right. I've been wanting to use this as an opportunity to clean this stuff up, but this one may not be so cut and dry.